extensions/wikia/SpecialDiscussionsLog/crowdin.conf

```diff
@@ -0,0 +1,5 @@
+[crowdin.project]
+projectid = "ext-SpecialDiscussionsLog"
+product_person = "Kornelia"
```

Comment:
put `kkydon` here because in other hand script will break while uploading

Reply:
Interesting, it didn't break for me. Will change anyway, thanks.